share/OpenMS/CHEMISTRY/Custom_RNA_modifications.tsv

```diff
@@ -0,0 +1,34 @@
+# This file contains OpenMS- and user-defined RNA modifications.
+# It is read in addition to the Modomics data file ("Modomics.tsv") and uses the same format (except for the optional "alternatives" column).
+# For terminal modifications, the "new_nomenclature" value has to end in "N" and contain either "33" (3'-terminal mod.) or "55" (5'-terminal mod.).
```

Comment:
why 33 and 55? is this an official nomenclature?

Reply:
Yep, Modomics nomenclature.